scanomatic/generics/abstract_model_factory.py

```diff
@@ -1030,7 +1029,7 @@ def serialize(self, model):
         if not self._has_section_head(model):
             raise ValueError("Need a section head for serialization")
 
-        with LinkerConfigParser(id=id(model)) as conf:
+        with LinkerConfigParser(id=id(model), allow_no_value=True) as conf:
```

Comment:
Other instances of `LinkerConfigParser` have this set. It doesn't work without it, so I guess it should be there..?

Reply:
I don't much remember what that linker config parser does, but some magic with references within configs. I guess we have to chance that allowing no value is acceptable.

Reply:
All other instances (I think) where it is used this is set to `True`. There might be a reason why not in this case, but... Hopefully that will become clear when we try to fix the integration tests.